virtcontainers/qemu.go

```diff
@@ -779,6 +780,11 @@ func (q *qemu) startSandbox(timeout int) error {
 		}
 	}
 
+	if err := label.SetProcessLabel(q.config.ProcessLabel); err != nil {
```

Comment:
I'm curious, why not do this in `startVM()` ?

Reply:
This needs to be done as late as possible, since all processes that are executed by kata-runtime after this call, will now run them with the SELinux label.  If those are fairly privileged processes, we do not want to run them under confinement or we end up giving a lot more privs to the KVMLauncher, then is necessary.

Reply:
Done